src/Functions/evalMLMethod.cpp

```diff
@@ -94,7 +94,40 @@ class FunctionEvalMLMethod : public IFunction
 
 REGISTER_FUNCTION(EvalMLMethod)
 {
-    factory.registerFunction<FunctionEvalMLMethod>();
+    FunctionDocumentation::Description description = R"(
+Applies a trained machine learning model to input features to generate predictions.
```

Comment:
We should somehow mention or link how the user can create a "machine learning model".